docs/DoxygenLayout.xml

```diff
@@ -3,7 +3,8 @@
   <!-- Navigation index tabs for HTML output -->
   <navindex>
     <tab type="mainpage" visible="yes" title="Introduction"></tab>
-    <tab type="modules" visible="yes" title="API Documentation" intro="These are the libraries supplied in the Raspberry Pi Pico SDK"/>
+    <tab type="topics" visible="yes" title="API Documentation" intro="These are the libraries supplied in the Raspberry Pi Pico SDK"/>
```

Comment:
Frustratingly, this change will break building the docs on Doxygen 1.8.17 as this seems to be a breaking change in Doxygen itself.

Reply:
Ah it's a README thing then. 'kay.